recipes/sanic-plugins-framework/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "0.9.4.post1" %}
+{% set name = "sanic-plugins-framework" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/Sanic-Plugins-Framework-{{ version }}.tar.gz
+  sha256: eea5b874c15255467c21acfb2e8b16b0e6bff5d11dffed698c63b2e759f3ec50
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
```

Comment:
If you use grayskull to create the recipe this will be updated to the latest syntax automatically. See https://github.com/conda-incubator/grayskull

Reply:
Firstly: Fantastic name for a tool. Secondly: Thank you for the tip - on it!

Reply:
Just used the tool - awesome!